pepper-apis/src/main/java/org/broadinstitute/ddp/service/PdfGenerationService.java

```diff
@@ -504,13 +498,13 @@ private int mergeAllProviderPdfsIntoOutput(ByteArrayOutputStream output, List<Me
      * Loads a font that is generally useful for writing out non-latin symbols
      */
     private PdfFont loadInternationalFont() {
-        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
-        byte[] freeSans;
         try {
-            freeSans = IOUtils.toByteArray(classLoader.getResourceAsStream(FONT_PATH));
-            return PdfFontFactory.createFont(freeSans, PdfEncodings.IDENTITY_H, true);
+            if (!PdfFontFactory.getRegisteredFonts().contains(INTERNATIONAL_FONT_NAME)) {
+                PdfFontFactory.register(INTERNATIONAL_FONT_PATH);
+            }
+            return PdfFontFactory.createRegisteredFont(INTERNATIONAL_FONT_NAME, PdfEncodings.IDENTITY_H, true);
         } catch (IOException e) {
```

Comment:
Changed this based on observation running unit tests locally: the font loading/creation is the most memory intensive part of running the pdf generation unit tests.
Don't think that overall this change improves memory situation: the change uses the itext font caching so it is not reloaded each time is needed. It MIGHT reduce the size of documents.